src/components/feature-board/content-elements/FileContentElementChips.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<div class="ma-2">
```

Comment:
I just did check the styling again the designs in Figma. Please use `padding` instead of a `margin` here.

```suggestion
	<div class="pa-2">
```

Reply:
I didn't know that you can look it up in Figma. Interesting! It won't make any visible difference at the moment. This would only be the case if we had a different background, but this might change sometime... So we will use padding.